apps/condo/domains/common/components/PostMessageProvider/types.ts

```diff
@@ -15,13 +15,15 @@ export const COMMON_ERROR_PREFIX = 'CondoWebAppCommonError' as const
 export type RequestParamsMap = {
     [Method in BridgeRequestMethodsName]: BridgeRequestParams<Method>
 } & {
-    CondoWebSendAnalyticsEvent: AnalyticsParams
+    CondoWebSendAnalyticsEvent: AnalyticsParams,
+    CondoWebSetActiveCall: { isActiveCall: boolean, connectedTickets: string[] }
```

Comment:
Let's be more consistent across the codebase
```suggestion
    CondoWebSetActiveCall: { isActiveCall: boolean, connectedTickets: Array<string> }
```